docs/integrations/sources/mongodb-v2.md

```diff
@@ -160,10 +160,28 @@ The MongoDB source utilizes change data capture (CDC) as a reliable way to keep
 
 Airbyte utilizes [the change streams feature](https://www.mongodb.com/docs/manual/changeStreams/) of a [MongoDB replica set](https://www.mongodb.com/docs/manual/replication/) to incrementally capture inserts, updates and deletes using a replication plugin. To learn more how Airbyte implements CDC, refer to [Change Data Capture (CDC)](https://docs.airbyte.com/understanding-airbyte/cdc/).
 
+### No Schema Mode
```

Comment:
is this the final name we settled on? I remember schema-less/packed being thrown around

Reply:
I don't think there's a term agreed upon.
"Wrapped mode" also suggested.

Title here is more of a description. not a name.
Does "Schema not enforced mode" sounds better?
"Schema Unenforced"?

Reply:
What's the name in the product? We should match the wording customers see in the UI.

Reply:
In the product you have a new "Schema Enforced" field
<img width="454" alt="image" src="https://github.com/airbytehq/airbyte/assets/867491/4b4d0a64-4160-43c2-80e5-35d520724ce9">


This sector is about when it's not enforced.
I'm taking the other comment from before and create a "Schema enforcement" section

Reply:
The new config field is "Schema Enforced"
<img width="454" alt="Screenshot 2023-12-18 at 11 56 58 AM" src="https://github.com/airbytehq/airbyte/assets/867491/6e4f1572-1a1f-4484-bdff-13598454e83c">